priv/schema/rabbitmq_web_mqtt.schema

```diff
@@ -0,0 +1,44 @@
+{mapping, "web_mqtt.num_acceptors.tcp", "rabbitmq_web_mqtt.num_tcp_acceptors", 
+    [{datatype, integer}]}.
+{mapping, "web_mqtt.num_acceptors.ssl", "rabbitmq_web_mqtt.num_ssl_acceptors", 
+    [{datatype, integer}]}.
+
+{mapping, "web_mqtt.tcp_config.port", "rabbitmq_web_mqtt.tcp_config.port",
```

Comment:
And what about `cowboy_opts` and `sockjs_opts`? Should "_opts" part be removed as well?